Readme.md

```diff
@@ -208,6 +208,7 @@ We follow these guidelines to produce legible code that adheres to the 79-charac
 - The contents of block expressions should be indented with 4 spaces relative to the start of the
   first and last lines.
 - Comma-separated arguments to a block parenthetical expression should all be on separate lines.
+- Any comma-seperated list over multiple lines should use a trailing comma
```

Comment:
Might want to clarify here or through additional examples that this should apply to literal lists, tuples, dictionaries, and sets, but also to function parameters.